ethereumj-core/src/main/java/org/ethereum/sharding/processing/validation/ValidationResult.java

```diff
@@ -27,6 +27,7 @@
     Exist,
     NoParent,
     StateMismatch,
+    Invalid,
```

Comment:
It should also be added to `ProcessingResult.fromValidation`. Looks like we need a note about that or better design to be sure that in future this problem won't arise again